src/interface/src/styleguide/filter-dropdown/filter-dropdown.component.ts

```diff
@@ -136,11 +136,13 @@ export class FilterDropdownComponent<T> implements OnInit {
     this.previousSelections = this.selectedItems.slice();
   }
 
-  clearSelections(e: Event): void {
+  clearSelections(e: Event, confirm?: boolean): void {
```

Comment:
Not a blocker, but I'm wondering if it might be simpler to have a separate confirmSelection() function.